src/uct/ib/mlx5/ib_mlx5.c

```diff
@@ -783,37 +784,31 @@ uct_ib_mlx5_select_sl(const uct_ib_iface_config_t *ib_config,
     ucs_config_sprintf_ulunits(sl_str, sizeof(sl_str), &ib_config->sl, NULL);
     ucs_config_sprintf_ternary(ar_enable_str, sizeof(ar_enable_str), &ar_enable,
                                NULL);
-    ucs_error("AR=%s was requested for SL=%s, but %s %s AR on %s,"
-              " SLs with AR support = { %s }, SLs without AR support = { %s }",
+    ucs_error("AR=%s was requested for SL=%s, but %s %s AR on "UCT_IB_IFACE_FMT
```

Comment:
i think better not use UCT_IB_IFACE_FMT without UCT_IB_IFACE_ARG

Reply:
fixed